src/app/config.service.ts

```diff
@@ -28,19 +28,40 @@ export class ConfigService {
    *
    * @return {boolean}
    */
-  useHttpdServer(): boolean {
-    const value = get(this.config, 'useHttpdServer');
+  useHttpServer(): boolean {
+    const value = get(this.config, 'contentSearch.enable');
     return isBoolean(value) ? value : value === 'true';
```

Comment:
You can compare with the operator ==
https://developer.mozilla.org/fr/docs/Web/JavaScript/Equality_comparisons_and_sameness#l%C3%A9galit%C3%A9_faible_avec